docs/AboutUs.md

```diff
@@ -1,30 +0,0 @@
-# About Us
```

Comment:
did I ? the file is still at the github. 
i just removed it from my staged files.